examples/lib/intermediateHtmlStepUi.js

```diff
@@ -2,9 +2,9 @@ var urlHash = require('./urlHash.js'),
   insertPreview = require('./insertPreview.js');
 
 function IntermediateHtmlStepUi(_sequencer, step, options) {
-  function stepUI() {
-    return '<div class="row insertDiv collapse">\
-          <section class="panel panel-primary .insert-step">\
+  function stepUI(id) {
+    return '<div class="row insertDiv collapse index' + id + '">\
```

Comment:
Here maybe index+id could be named a bit more specifically?

Reply:
So basically, I am using id as a variable and index just to signify its meaning for the code readability.